pymtl3/datatypes/PythonBits.py

```diff
@@ -478,7 +478,7 @@ def __bool__( self ):
     return self._uint != 0
 
   def __int__( self ):
-    return self._uint
+    return int(self._uint)
```

Comment:
Probably not too relevant but I have wondered (for some time) what is the type of `Bits._uint` after elaboration... I remember seeing code assuming it is an `int` but not always.. this is probably where explicit type annotations can help.

Reply:
It is kinda weird but this int casting is to cast a bool to int